.github/workflows/issues_to_project.yml

```diff
@@ -0,0 +1,25 @@
+name: Add issue/PR to AutoAgora project board
+on:
+  issues:
+    types:
+      - opened
+  pull_request:
+    types:
+      - opened
```

Comment:
I think only issues should trigger this workflow as we link them in to PRs.

Reply:
My view is that PRs also have to be tracked in the project board, assigned for review / merge to people, etc.

Reply:
I'm with @aasseman here. I think PRs should be on the board so that we track everything in one place.

Reply:
hey, our boards contain all kinds of tasks, not only issues/PRs, let's enable this for all kind of them